_data/akshat-max/akshat-max.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Akshat Mittal
+institution: Galgotias university
+profile_pic: Akshat.png
```

Comment:
If the file path is case-sensitive, then the ```A``` needs to be lowercase.

Reply:
Thank you for the help😀😀

Reply:
No problem! 😄